recipe/cross-linux.cmake

```diff
@@ -0,0 +1,18 @@
+# https://github.com/ContinuumIO/anaconda-issues/issues/8779
+# https://github.com/AnacondaRecipes/freeglut-feedstock/blob/master/recipe/cross-linux.cmake
+# this one is important
```

Comment:
These notes could be a bit more informative.

Reply:
Addressed.